pkgs/top-level/all-packages.nix

```diff
@@ -9780,7 +9780,7 @@ in
 
   pprof = callPackage ../development/tools/profiling/pprof { };
 
-  pyprof2calltree = pythonPackages.callPackage ../development/tools/profiling/pyprof2calltree { };
+  pyprof2calltree = throw "pyprof2calltree can now be found in pythonPackages";
```

Comment:
Don't do this, we still want the top-level attribute `pyprof2calltree`.
Instead do:
```
  pyprof2calltree = with python3Packages; toPythonApplication pyprof2calltree;
```

Reply:
Changed.